syntax="proto3";
package cmsg;
message CMsgHead{
  int32 msg_size = 1;
  string msg_str = 2;
  int32 msg_type = 3;
}
	 